src/com/dojo/foundations/parking/TheGarage.java

```diff
@@ -218,54 +166,74 @@ public static void main(String[] args) throws Exception {
+            new Cell("C8", CellTypeEnum.CAR, true, Optional.empty()),
+            new Cell("T1", CellTypeEnum.TRUCK, true, Optional.empty()),
+            new Cell("T2", CellTypeEnum.TRUCK, true, Optional.empty()),
+            new Cell("T3", CellTypeEnum.TRUCK, true, Optional.empty()),
+            new Cell("T4", CellTypeEnum.TRUCK, true, Optional.empty())
+        ));
+    }
+    
+    private static void printGarageMenu() {
+        System.out.println("\"The Garage\"");
+        System.out.println("1) Motorcycle enters");
+        System.out.println("2) Car enters");
+        System.out.println("3) Truck enters");
+        System.out.println("4) Motorcycle leaves ");
+        System.out.println("5) Car leaves");
+        System.out.println("6) Truck leaves");
+        System.out.println("7) Get Parking lot occupation");
+        System.out.println("8) Get free cells");
+        System.out.println("0) CLOSE");
+    }
+
     private static boolean isValidInput(int option) {
-        if (option < 0 || option > 6) {
+        if (option < 0 || option > 8) {
             System.out.println("Wrong selection, please try again");
             return false;
         }
         return true;
     }
 
-    private static double calculateBillMotorcycle(Cell cell, Motorcycle motorcycle) {
-        double hours = (double) ChronoUnit.MINUTES
-                .between(cell.getEnterDateTime().get(), LocalDateTime.now()) / 60;
-        double completeHours = Math.floor(hours);
-        double incompleteHours = hours - completeHours;
-        int hoursToCharge = (incompleteHours <= WAIVE_TIME) ? (int) completeHours: (int) Math.ceil(hours);
-        return (motorcycle.cc < 200) ?
-                hoursToCharge * RateEnum.MOTORCYCLE.getRate() :
-                hoursToCharge * RateEnum.MOTORCYCLE.getRate() * HIGH_CC_OVERCHARGE;
-    }
-
-    private static double calculateBillCar(Cell cell, Car car) {
-        double hours = (double) ChronoUnit.MINUTES
-                .between(cell.getEnterDateTime().get(), LocalDateTime.now()) / 60;
-        double completeHours = Math.floor(hours);
-        double incompleteHours = hours - completeHours;
-        int hoursToCharge = incompleteHours <= WAIVE_TIME ? (int) completeHours: (int) Math.ceil(hours);
-        return hoursToCharge * RateEnum.CAR.getRate();
-    }
-
-    private static double calculateBillTruck(Cell cell, Truck truck) {
-        double hours = (double) ChronoUnit.MINUTES
-                .between(cell.getEnterDateTime().get(), LocalDateTime.now()) / 60;
-        double completeHours = Math.floor(hours);
-        double incompleteHours = hours - completeHours;
-        int hoursToCharge = incompleteHours <= WAIVE_TIME ? (int) completeHours: (int) Math.ceil(hours);
-        return (truck.ton < 4.0) ?
-                hoursToCharge * RateEnum.TRUCK.getRate() :
-                hoursToCharge * RateEnum.TRUCK.getRate() * HIGH_TON_OVERCHARGE;
+    private static void evaluatePlate(Matcher matcher) {
```

Comment:
Do this method do the job? It is a very good idea and necessary to create this private method, but do the 'return' clause have the desired effect of escaping the execution in the main loop when the plate is not correct?